components/card/stories/card.stories.js

```diff
@@ -68,10 +73,39 @@ export default {
 			control: { type: "text" },
 		},
 		footer: { table: { disable: true } },
-		isGallery: { table: { disable: true } },
-		isCardAssetOverride: { table: { disable: true } },
+		isGallery: {
+			name: "Gallery preview",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "Component",
+			},
+			control: "boolean",
+			if: { arg: "isHorizontal", truthy: false },
+		},
+		isCardAssetOverride: {
```

Comment:
Override feels a touch vague for what this is actually doing. Is there a more descriptive way to describe the change such as "Image fill" or "Gallery style"?

Reply:
What do you think of `isImageFill`? Gallery already has a control, with different styles, so I went with `isImageFill`. That did require a change to args in asset, however. https://github.com/adobe/spectrum-css/pull/3443/commits/874f6cc8d674b719fa090bf3f83581d0e7825c5d